client/my-sites/posts/main.jsx

```diff
@@ -73,14 +80,19 @@ const PostsMain = React.createClass( {
 					siteId={ site.ID }
 					query={ this.props.draftsQuery } />
 				<QueryPostCounts siteId={ site.ID } type="post" />
-				<Card compact className="posts__drafts-header">
-					{ this.translate( 'Latest Drafts' ) }
-					<Button borderless href={ this.props.newPostPath }>
-						<Gridicon icon="plus" />
+				<SectionHeader className="posts__drafts-header" label={ this.translate( 'Latest Drafts' ) }>
```

Comment:
I guess same as it is currently, but seems strange we render the header when there are no drafts on the site currently?